scripts/west_commands/runners/core.py

```diff
@@ -140,9 +139,7 @@ def get(self, option, *args):
 
     def _init(self):
         build_z = os.path.join(self.build_dir, 'zephyr')
-        generated = os.path.join(build_z, 'include', 'generated')
-        files = [os.path.join(build_z, '.config'),
-                 os.path.join(generated, 'generated_dts_board.conf')]
+        files = [os.path.join(build_z, '.config')]
         for f in files:
             self._parse(f)
```

Comment:
Might as well simplify this entire function down to this:

```python
self._parse(os.path.join(self.build_dir, 'zephyr', '.config'))
```

At that point, `_init()` could probably be removed as well, and the code moved into the caller.

Reply:
Done.